daemon/src/shared_protocol.rs

```diff
@@ -13,7 +13,10 @@ use maia_core::secp256k1_zkp::XOnlyPublicKey;
 use maia_core::PartyParams;
 use std::ops::RangeInclusive;
 use std::time::Duration;
+use tracing::instrument;
+use tracing::Instrument;
 
+#[instrument]
```

Comment:
If we only do `#[instrument]` it will include all the arguments right? I'm not sure if we need this information but at the very least we should exclude `own_cets` and `counterparty_cets`. That will be a significant amount of data.

Reply:
Thanks, good point! I'll circlr back to the others too and make sure it isn't logging too much.